static_src/main.js

```diff
@@ -93,7 +92,7 @@ function app(orgGuid, spaceGuid, appGuid) {
   appActions.fetch(appGuid);
   appActions.fetchStats(appGuid);
   ReactDOM.render(
-    <MainContainer initialSpaceGuid={ spaceGuid }>
+    <MainContainer>
       <AppContainer
         initialAppGuid={ appGuid }
```

Comment:
Shouldn't AppStore also follow this pattern and maintain the current application guid with an action to change it?